{"uuid": "109b7387-bbcf-43cd-b8cf-a1c11bdb39da", "children": ["fd8ef618-3cf7-4f82-b6fa-8f282637d771", "d6496cc9-2d88-48d3-8911-a145e2ee2792"], "befores": [{"name": "create_token", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='restful-booker.herokuapp.com', port=443): Max retries exceeded with url: /auth (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000161B6C341A0>: Failed to resolve 'restful-booker.herokuapp.com' ([Errno 11001] getaddrinfo failed)\"))\n", "trace": "  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 897, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\conftest.py\", line 14, in create_token\n    response = post_request(\n               ^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\src\\helpers\\api_requests_wrapper.py\", line 13, in post_request\n    post_response = requests.post(url=url, headers=headers, auth=auth, data=json.dumps(payload))\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\requests\\api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\requests\\api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\requests\\sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\requests\\sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ViMS\\PycharmProjects\\PyAPIAutomationFramework\\.venv\\Lib\\site-packages\\requests\\adapters.py\", line 700, in send\n    raise ConnectionError(e, request=request)\n"}, "start": 1723733656810, "stop": 1723733656817}], "start": 1723733656810, "stop": 1723733657282}